src/glue/jobs/s3_to_json_s3.py

```diff
@@ -167,32 +167,36 @@ def get_json_schema(archive_map, file_metadata, json_schemas):
             "archive_map_version": os.environ.get("archive_map_version")
     }
     file_metadata["assessment_revision"] = int(file_metadata["assessment_revision"])
+    valid_assessments = []
     for assessment in archive_map["assessments"]:
```

Comment:
Nit: It'd be nice if there was a comment on what this for loop is doing.

Reply:
This function interprets archive-map.json. So you should reference [the archive-map.json schema](https://github.com/Sage-Bionetworks/mobile-client-json/blob/main/schemas/v3/ArchiveMap.json) to understand what's going on.

I'll add a comment.